### ISSUE TITLE
Add multiple JS files to a view in only one minifyed JS file.

### ISSUE THREAD

**Issue by Abuelodelanada** [salesagility/SuiteCRM#508]
The most common way to add JS files to a view is through metadata files, such as editviewdefs.php.

![editviewdefs_accounts](https://cloud.githubusercontent.com/assets/939888/10172007/67056b04-66b3-11e5-8ef8-eb614bedade6.png)

This way is simple, but has two problems:
- The application makes many requests as files included.
- Each file is not minifyed.

This produces unnecessary overload to the webserver and network.

We should implement a way to load multiple JS files from the constructor of the view, and the application should automatically join and minify all these files.
This will produce only one JS request per view

![view edit php_accounts](https://cloud.githubusercontent.com/assets/939888/10172804/a3c35dcc-66b7-11e5-9a38-ee64bc3e3fa2.png)

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Add multiple JS files to a view in only one minifyed JS file.](https://trello.com/c/4Fnb2OhF/146-add-multiple-js-files-to-a-view-in-only-one-minifyed-js-file)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
